stories/index.js

```diff
@@ -16,6 +16,15 @@ storiesOf('Button', module)
     })
     .add('default', () => <Button text="Hello World!" />);
 
+storiesOf('Image', module).addParameters({
+    backgrounds: [
+        { name: 'light', value: '#fffef0' },
+        { name: 'dark', value: '#1e3b47' },
+        { name: 'purple', value: '#7955ff', default: true }
+    ],
+}).add('default', () => (
+    <Image src="https://i.pinimg.com/originals/aa/49/54/aa4954451f16847cb657e68e5f46538a.jpg"/>
```

Comment:
Maybe add local image instead?

Reply:
Adding the image locally will be better.